xcode/Subconscious/Shared/Components/Detail/MemoViewerDetailView.swift

```diff
@@ -140,6 +142,22 @@ struct MemoViewerDetailLoadedView: View {
             )
         )
     }
+    
+    private func onLink(
+        url: URL
+    ) -> Bool {
+        guard let sub = url.toSubSlashlinkURL() else {
+            return true
+        }
+        notify(
+            .requestFindDetail(
+                slashlink: sub.slashlink,
+                traversalPath: traversalPath,
+                fallback: sub.fallback
+            )
+        )
+        return false
+    }
```

Comment:
Enables link taps in the viewer, we can drop this if it conflicts with the upcoming viewer improvements.